helm/values-prod.yaml

```diff
@@ -28,7 +28,3 @@ rollout:
         - pause: {}
         - setWeight: 40
         - pause: {}
-        - setWeight: 60
-        - pause: { duration: 10 }
-        - setWeight: 80
-        - pause: { duration: 10 }
```

Comment:
remove extra waits for deployment as they are maddening. At this point we've opt'd in to a 20 and 40 percent roll out... that's enough